### DISCUSSION TITLE
File upload fails after upgrading Livewire to v3

### DISCUSSION THREAD

**Discussion by NikarashiHatsu** [livewire/livewire#6775] [1. Help]
After using Livewire's Upgrade Tool, my profile picture upload component fails without any clear trace to debug. It simply throws `The file failed to upload`.

So my questions are:
1. What file(s) I should check?
2. The browser throws 422 error code, which has an indication that my uploaded file is failed to pass the validation, but the validation itself comply the file upload. Is there any Livewire validation did in the background before it is uploaded to the storage?
3. The file I upload cannot exceed 1MB, even though I set the current rule to 8MB. Which file did the validation before in-component validation?

Thanks for advance, any help is appreciated.

[UPDATE]
I did create new project, fresh Laravel installation, required the Livewire, and still it cannot upload file exceeding 1MB of size.

[UPDATE 2]
Apparently my PHP was corrupted(?), so I reinstall PHP completely, and it works...somehow.

**Comment by norouzimohammadreza** [comment]
can you help me about this problem?
